azure-pipelines.yml

```diff
@@ -77,6 +77,14 @@ steps:
     cargo test --verbose --all --features "test unstable"
   displayName: test
 
+- script: |
+    cargo install --verbose cargo-tarpaulin --version '0.13.3'
+    cargo tarpaulin --verbose --all --features "test unstable" --out Xml
+    bash <(curl -s https://codecov.io/bash)
+  displayName: cov
+  env:
+    CODECOV_TOKEN: $(CODECOV_TOKEN_SECRET)
```

Comment:
```suggestion
```

My understanding is that for public repos we shouldn't need a token, so let's try omitting this? I've enabled codecov from the web interface.

Source: https://devblogs.microsoft.com/devops/uploading-to-codecov-just-got-easier/